versions-common/src/test/java/org/codehaus/mojo/versions/api/ArtifactVersionsTest.java

```diff
@@ -64,21 +64,22 @@ public void test4DigitVersions() throws Exception {
         assertEquals("artifact", instance.getArtifactId());
         assertEquals("group", instance.getGroupId());
         assertThat(
-                instance.getVersions(),
+                instance.getVersions(true),
                 arrayContaining(versions("1.0.0.1", "1.0.0.2", "1.1.1", "2.100.0.1", "2.121.2.1", "3.1.0.1")));
```

Comment:
test for `false` value will be appreciated

Reply:
Added